.eslintrc.json

```diff
@@ -0,0 +1,28 @@
+{
+  "root": true,
+  "extends": [
+    "eslint:recommended",
+    "plugin:import/errors",
+    "plugin:import/warnings"
```

Comment:
I would expect the import plugin to be listed in plugs. Is it inherited by another?

Reply:
Could not trace where exactly it is inherited from but, it is.